color_helper.sublime-settings

```diff
@@ -338,6 +338,18 @@
                 "meta.tag.any.html meta.attribute-with-value.style.html"
             ]
         },
+        {
+            "name": "JavaScript/JSX",
+            "base_scopes": [
+                "source.js",
+                "source.jsx",
```

Comment:
Tests reported dangling comma. Last item in the list should not have a comma  (for proper JSON, though Sublime will parse it fine). So let's just remove that last comma.